### ISSUE TITLE
AI playTurn doesn't refresh the screen correctly

### ISSUE THREAD

**Issue by ideascape** [Hansigzandrinovka/BlackJack368Project#14]
The desired behavior is, each time a card is drawn, we see that card appear on the screen. It should be a discrete step for each card.

Two things: first test to see if each drawn card is displayed with its own addRefresh call, instead of the entire turn. This may already be working, it wasn't easy for me to tell, since the AIs don't hit very often. Maybe re-enable a bad AI that draws a bunch of cards to make sure things are working right.

Second, the screen goes blank at some point during each AIs turn. This shouldn't happen. Every progression should be smooth, as with the drawCards() stage of the game.

**Comment by mbechtel2** [comment]
I think the problem was that addRefresh called to the game's playerArray and pot, but the Player class didn't have access to either of them. I made it to where the Player class takes the entire game object as a parameter rather than the deck (but I set theDeck equal to the gameObject's deck so no problems would happen) which partially fixed the AI's turns. There is still a problem of the finished hands partially flashing prior to the actual dealing so I not going to close this issue quite yet.
